server/src/main/java/org/solarbank/server/dto/ErrorResponse.java

```diff
@@ -0,0 +1,32 @@
+package org.solarbank.server.dto;
+
+import lombok.Data;
+
+@Data
+public class ErrorResponse {
+    private ErrorDetails error;
+
+    public ErrorResponse(int code, String status, String message) {
+        this.error = new ErrorDetails(code, status, message);
+    }
+
+    @Data
+    public static class ErrorDetails {
+        private int code;
+        private String status;
+        private String message;
+
+        /**
```

Comment:
If your happy to remove the docstrings, here is another one to get rid of.

Reply:
docstring removed